Vungle/src/main/java/com/mopub/mobileads/VungleBanner.java

```diff
@@ -320,11 +326,11 @@ public void onAdStart(@NonNull String placementReferenceId) {
             if (mPlacementId.equals(placementReferenceId)) {
                 mIsPlaying = true;
                 //Let's load it again to mimic auto-cache
-                if (AdSize.isBannerAdSize(mAdConfig.getAdSize())) {
-                    sVungleRouter.loadBannerAd(mPlacementId, mAdConfig.getAdSize(), mVungleRouterListener);
-                } else if (VUNGLE_MREC == mAdConfig.getAdSize()) {
-                    sVungleRouter.loadAdForPlacement(mPlacementId, mVungleRouterListener);
-                }
+//TODO                if (AdSize.isBannerAdSize(mAdConfig.getAdSize())) {
+//                    sVungleRouter.loadBannerAd(mPlacementId, mAdConfig.getAdSize(), mVungleRouterListener);
```

Comment:
old code should be fine however, I'm not sure why we even pass mVungleRouterListener. Even if it fails we should not have generated the error

Reply:
Hi @alexander-gubov, during e2e testing of adMarkup, I got the following exception:
```
2021-02-23 08:18:49.045 19744-19999/com.vungle.mopubtest E/com.vungle.warren.AdLoader: Failed to load Ad/Assets for AdRequest{placementId='MREC-1572558', adMarkup=null, type=0, adCount=1}. Cause : 
    com.vungle.warren.error.VungleException: No Event id passed for HBP
        at com.vungle.warren.AdLoader$4.run(AdLoader.java:627)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:923)
```

So considering if we should remove the logic at onAdStart() callback for the purpose of mimic auto-cache.

Reply:
keep the original logic as before, due to for non-HB enabled placements.

Reply:
I mean even non-hb banner loading can fail, we should not generate any error for already displaying banner, " No Event id passed for HBP" in the logs should be fine for HB